src/transaction.js

```diff
@@ -600,6 +614,35 @@ const transaction = {
     }
   },
 
+  /**
+   * Execute some verifications if this data can become a transaction
+   * Validates maximum number of inputs and outputs
```

Comment:
This docstring (the first sentence) doesn't help much. I would suggest:

```
Validate transaction information. For now, we only verify the maximum number of inputs and outputs.
```

Reply:
Yeah, I agree it's better. Updated it, thanks.